sky/data/data_utils.py

```diff
@@ -225,3 +328,123 @@ def run_upload_cli(command: str, access_denied_message: str, bucket_name: str):
                 raise exceptions.StorageUploadError(
                     f'Upload to bucket failed for store {bucket_name}. '
                     'Please check the logs.')
+
+
+def store_rclone_config(bucket_name: str,
+                        region: str,
+                        apply_changes: bool = True):
+    """creates a configuration files for rclone - used for
+    bucket syncing and mounting """
+    # pylint: disable=import-outside-toplevel
+    import textwrap
+
+    rclone_file = os.path.expanduser(RCLONE_CONFIG_PATH)
+    access_key_id, secret_access_key = ibm.get_hmac_keys()
+
+    config_data = textwrap.dedent(f"""\
+        [{bucket_name}]
+        type = s3
+        provider = IBMCOS
+        access_key_id = {access_key_id}
+        secret_access_key = {secret_access_key}
+        region = {region}
+        endpoint = s3.{region}.cloud-object-storage.appdomain.cloud
+        location_constraint = {region}-smart
+        acl = private
+        """)
+
+    # create ~/.config/rclone if doesn't exist
+    if not os.path.isdir(os.path.dirname(rclone_file)):
+        os.makedirs(os.path.dirname(rclone_file), exist_ok=True)
+    # create rclone.conf if doesn't exist
+    if not os.path.isfile(rclone_file):
+        open('file', 'w').close()
```

Comment:
```suggestion
    rclone_config_dir_path = os.path.dirname(rclone_config_path)
    if not os.path.isdir(rclone_config_dir_path):
        os.makedirs(rclone_config_dir_path, exist_ok=True)
    # create rclone.conf if doesn't exist
    if not os.path.isfile(rclone_config_path):
        open('file', 'w').close()
```

Reply:
made it even simpler since: 
```
    # create ~/.config/rclone/ if doesn't exist
    os.makedirs(os.path.dirname(rclone_config_path), exist_ok=True)
    # create rclone.conf if doesn't exist
    if not os.path.isfile(rclone_config_path):
        open('file', 'w').close()
```
Can't find a justification for a single use variable. Readability is guaranteed via comments.